cloudsplaining/shared/multi-account-config.yml

```diff
@@ -0,0 +1,4 @@
+accounts:
+  default_account: 123456789012
+  prod: 123456789013
+  test: 123456789014
```

Comment:
❓  Does this sample content need to be added to `constants.py` and then referenced in `create_multi_account_config_file.py` instead of `EXCLUSIONS_TEMPLATE`?

Reply:
Great catch. Can't believe I missed that lol. I fixed this, and added a click unit test to validate that for the hell of it.